modules/update-service-create-service-web-console.adoc

```diff
@@ -0,0 +1,35 @@
+[id="update-service-create-service-web-console_{context}"]
+= Creating an OpenShift Update Service application by using the web console
+
+You can use the {product-title} web console to create an Update Service application by using the OpenShift Update Service Operator.
+
+.Prerequisites
```

Comment:
The current and target releases have been mirrored to the internal registry is also a prerequisite

Reply:
done